<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registerwindow</class>
 <widget class="QMainWindow" name="registerwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_irparalogin">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>300</y>
      <width>277</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Ir para Login</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>321</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_email_3">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Nome: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_nome">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(164, 249, 203);
border-radius: 2px;
color:black;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_email">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>E-mail: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_email">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(164, 249, 203);
border-radius: 2px;
color:black;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_senha">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Senha: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_senha">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(164, 249, 203);
border-radius: 2px;
color:black;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::EchoMode::Password</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_registro">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Registrar</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_login">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>341</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:26pt; font-weight:700;&quot;&gt;Registro&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>91</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/resource/resource/logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
